Logger.Tests/BaseLoggerMixinsTests.cs

```diff
@@ -26,14 +26,30 @@ public void Error_WithData_LogsMessage()
             var logger = new TestLogger();
 
             // Act
-            //logger.Error("Message {0}", 42);
+            logger.Error("Message {0}", 42);
 
             // Assert
             Assert.AreEqual(1, logger.LoggedMessages.Count);
             Assert.AreEqual(LogLevel.Error, logger.LoggedMessages[0].LogLevel);
             Assert.AreEqual("Message 42", logger.LoggedMessages[0].Message);
         }
 
+
+        [TestMethod]
+        public void TestError_WithMessageArgs()
+        {
+            string path = Path.GetTempFileName();
+            if (File.Exists(path))
+                File.Delete(path);
+            FileLogger logger = new FileLogger(path);
+            logger.Error("Error_Message_{0}_{1}_{2}", 1, 2, 3);
+            DateTime expectedDate = DateTime.Now;
+            StreamReader sr = File.OpenText(path);
+            Assert.AreEqual($"{expectedDate} FileLogger Error: Error_Message_1_2_3", sr.ReadLine());
+            sr.Close();
+            File.Delete(path);
+        }
```

Comment:
I would recommend using the Arrange Act Assert structure for tests with more lines of code like this to make them easier to read.
```suggestion
        [TestMethod]
        public void TestError_WithMessageArgs()
        {
            // Arrange
            string path = Path.GetTempFileName();
            if (File.Exists(path))
                File.Delete(path);
            FileLogger logger = new FileLogger(path);
            logger.Error("Error_Message_{0}_{1}_{2}", 1, 2, 3);
            DateTime expectedDate = DateTime.Now;
            
            // Act
            StreamReader sr = File.OpenText(path);
            string actualContents = sr.ReadLine();
            
            // Assert
            Assert.AreEqual($"{expectedDate} FileLogger Error: Error_Message_1_2_3", actualContents);
            sr.Close();
            File.Delete(path);
        }
```